templates/post_note.tpl.html

```diff
@@ -64,12 +64,21 @@
     $().ready(function() {
         $('#clear_note_cc').click(clearExtraRecipients);
         $('#add_email_signature').click(setSignature);
+
+        $('#post_note_form').garlic({
+            getPath: function ( $elem ) {
+              var form = $elem.parents('form');
+              return form.find('input[name=issue_id]').val() + '_' +
+                      form.find('input[name=parent_id]').val() +
+                      form.data('sup-id') + $elem.attr( 'name' );
+            }
```

Comment:
i think should always specify the separator between elements to avoid same id for:
- parent_id 1 + sup_id 11
- parent_id 11 + sup_id 1

[ item1, item2 ].join('_') ?